spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -41,7 +59,15 @@ def self.statuses; end
       end
 
       it "rejects a record where the attribute is not defined as an enum with should not" do
-        message = "Did not expect #{record_with_array_values.class} to define :#{enum_attribute} as an enum"
+        message = "Did not expect #{record_with_array_values.class} to define :#{enum_attribute} as an enum and store the value in a column with an integer type"
+
+        expect do
+          expect(record_with_array_values).to_not define_enum_for(enum_attribute)
+        end.to fail_with_message(message)
+      end
+
+      it "rejects a record where the saved attribute does not come back out after saving" do
```

Comment:
Line is too long. [92/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.